llvm/include/llvm/CodeGen/MacroFusion.h

```diff
@@ -48,15 +48,15 @@ bool fuseInstructionPair(ScheduleDAGInstrs &DAG, SUnit &FirstSU,
 
 /// Create a DAG scheduling mutation to pair instructions back to back
 /// for instructions that benefit according to the target-specific
-/// shouldScheduleAdjacent predicate function.
+/// predicate functions.
```

Comment:
Please document that shouldScheduleAdjacent will be true if any of the provided predicates are true